regolith/builders/recentcollabsbuilder.py

```diff
@@ -125,16 +126,17 @@ def latex(self):
             emp = p.get("employment", [{"organization": "missing",
                                         "begin_year": 2019,}])
             emp.sort(key=ene_date_key, reverse=True)
-            if p.get("employment", "position") == "editor":
-                coeditors = p.get("employment", "co-workers")
+            if emp.get("coediters") != None:
+                my_coediters = emp.get("coediters")
```

Comment:
I think editor is spelled editor not editer.

Also, I think in the database we want it to be ``coworkers`` because if your employment is as a software engineer, or as an undergrad, or pretty much anything other than an editor, having a coeditor field doesn't make too much sense.   So the logic I thought we discussed was that we would have a ``coworkers`` field, but we would only load it into my_coeditors if the employment type was editor.